srcpkgs/easystroke/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'easystroke'
+pkgname=easystroke
+version=0.6.0
+revision=1
+archs="i686 x86_64"
+wrksrc=patched-easystroke-master
+build_wrksrc=easystroke
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="boost-devel dbus-glib-devel gtkmm-devel xorg-server-devel"
+short_desc="Gesture recognition application for X11"
+maintainer="lopid <github23429@rainslide.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/thjaeger/easystroke/wiki/"
+distfiles="http://openartisthq.org/easystroke/patched-easystroke-master.tar.bz2"
+checksum=18bf167eaf7c41d91e6c17b1f8436eeb0988d34d47f523ef03789caab81e902a
+tags="mouse gestures"
+
+do_build() {
+	make PREFIX=/usr
```

Comment:
Add `make_build_args="PREFIX=/usr"` to template instead.

Reply:
It won't build with that. Here's the difference, with your suggestion on the left: 
![Screenshot_2021-07-25_15-00-16](https://user-images.githubusercontent.com/87655256/126901922-850f26a7-01c1-43dc-8b9a-ff91bdee0b9a.png)